help/sources/tutorials/api/create/crm/sugarcrm-accounts-contacts.md

```diff
@@ -0,0 +1,965 @@
+                "custom_score_field": "0",
+                "deliverability_status": 0
+            },
+            "page_size": 100
+        },
+        {
+            "next": "https://developer.salesfusion.com/api/2.0/contacts/?date_field=updated_date&page=2",
+            "page_number": 1,
+            "total_count": 135704,
+            "count": 135704,
+            "results": {
+                "contact_id": 2,
+                "contact": "https://developer.salesfusion.com/api/2.0/contacts/2/",
+                "email": "test.user@outlook.com",
+                "created_date": "2022-06-21T23:35:00Z",
+                "updated_date": "2022-02-03T21:55:00Z",
+                "opt_out": "N",
+                "crm_type": "Contact",
+                "deliverability_status": 0
+            },
+            "page_size": 100
+        },
+        {
+            "next": "https://developer.salesfusion.com/api/2.0/contacts/?date_field=updated_date&page=2",
+            "page_number": 1,
+            "total_count": 135704,
+            "count": 135704,
+            "results": {
+                "contact_id": 3,
+                "contact": "https://developer.salesfusion.com/api/2.0/contacts/3/",
+                "first_name": "Jonathan",
+                "last_name": "Ryan",
+                "phone": "+1 - 000 - 000 - 0000",
+                "email": "test.user@yahoo.com",
+                "account_id": 52,
+                "account_name": "Q3 ARVRO III PR",
+                "account": "https://developer.salesfusion.com/api/2.0/accounts/52/",
+                "owner_name": "Max Jensen",
+                "owner": "https://developer.salesfusion.com/api/2.0/users/45/",
+                "created_date": "2022-07-02T23:35:00Z",
+                "created_by": "https://developer.salesfusion.com/api/2.0/users/45/",
+                "updated_date": "2022-02-03T21:55:00Z",
+                "updated_by": "https://developer.salesfusion.com/api/2.0/users/45/",
+                "created_by_id": 45,
+                "updated_by_id": 45,
+                "crm_id": "f54a1840-84ea-11ec-9546-06a1e215b98e",
+                "opt_out": "N",
+                "crm_type": "Lead",
+                "status": "New",
+                "title": "Director Sales",
+                "custom_score_field": "0",
+                "deliverability_status": 0
+            },
+            "page_size": 100
+    ]
+}
+```
+
+### Create a source connection {#source-connection}
+
+You can create a source connection by making a POST request to the [!DNL Flow Service] API. A source connection consists of a connection ID, a path to the source data file, and a connection spec ID.
+
+**API format**
+
+```https
+POST /sourceConnections
+```
+
+**Request**
+
+The following request creates a source connection for [!DNL SugarCRM Accounts & Contacts]:
+
+If you need to work with the [!DNL SugarCRM] Accounts API the `object_type` property value should be `accounts`.
+
```

Comment:
We can circumvent the two examples by using tabs. Essentially, right before the shell example, we can add:

`>[!BEGIN TABS]`

`>[!TAB Accounts]`

*request example here*

`>[!TAB Contacts]`

*request example here*

`>[!ENDTABS]`

Then we can just repeat the same for response. Doc here for additional guidance: https://experienceleague.adobe.com/docs/authoring-guide-exl/using/markdown/syntax-style-guide.html?lang=en#tabs

Reply:
We can apply the same to the explore examples as well

Reply:
Introduced on both the UI and API pages for the SugarCRM Accounts & Contacts connector.